serverless/service/plugins/deployment_bucket.py

```diff
@@ -0,0 +1,55 @@
+from typing import Dict, Optional
+
+from serverless.service.plugins.generic import Generic
+
+
+class InvalidInputException(Exception):
+    pass
+
+
+class DeploymentBucket(Generic):
```

Comment:
Please add link to the plugin, i.e. https://www.npmjs.com/package/serverless-deployment-bucket in doc e.g. https://github.com/epsylabs/serverless-builder/blob/master/serverless/service/plugins/vpc_discovery.py#L6